omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -810,9 +814,7 @@ def generate_config(node_name)
 
       generate_config_for_topology(PrivateChef["topology"], node_name)
 
-      unless PrivateChef["ldap"].nil? || PrivateChef["ldap"].empty?
-        gen_ldap
-      end
+      gen_ldap if PrivateChef["ldap"]["enabled"]
```

Comment:
We wouldn't want to use `ldap_authentication_enabled?` here? or where is that fun being use? 🤔